class GameOverException(Exception):
    pass


class PuzzleSolvedException(Exception):
    pass
